diff --git a/IdeaVIM/build.gradle b/IdeaVIM/build.gradle
index 80310f93..def8f834 100644
--- a/IdeaVIM/build.gradle
+++ b/IdeaVIM/build.gradle
@@ -9,7 +9,7 @@ buildscript {
 }

 plugins {
-  id 'org.jetbrains.intellij' version '0.4.9'
+  id 'org.jetbrains.intellij'
 }

 apply plugin: 'java'
diff --git a/AceJump/build.gradle.kts b/AceJump/build.gradle.kts
index b751836..92ec6df 100755
--- a/AceJump/build.gradle.kts
+++ b/AceJump/build.gradle.kts
@@ -4,8 +4,8 @@ import org.jetbrains.intellij.tasks.PatchPluginXmlTask

 plugins {
   idea apply true
-  kotlin("jvm") version "1.3.61"
-  id("org.jetbrains.intellij") version "0.4.14"
+  kotlin("jvm")
+  id("org.jetbrains.intellij")
 }

 fun fetchChangeNotes() = File("CHANGES.md").readLines().drop(4).takeWhile { !it.startsWith("###") }.let { notes ->
